src/Framework/App.Runtime/src/Microsoft.AspNetCore.App.Runtime.csproj

```diff
@@ -474,6 +468,8 @@ This package is an internal implementation of the .NET Core SDK and is not meant
       PrivateUri="$(DotNetRuntimePrivateDownloadUrl)"
       PrivateUriSuffix="$(DotNetAssetRootAccessTokenSuffix)"
       DestinationPath="$(DotNetRuntimeArchive)" />
+    <RemoveDir Directories="$(RedistSharedFrameworkLayoutRoot)" />
```

Comment:
This seems weird, why remove then re-create? Add a comment?

Reply:
The `<MakeDir />` task isn't absolutely required on Windows because the `<Unzip />` task (like `<Copy />` and `<WriteLinesToFile />` will create target directories. But, the `tar` command fails miserably if the target directory doesn't exist.

So, remove the `<RemoveDir />` task and files may be leftover in `$(RedistSharedFrameworkLayoutRoot)`. Remove the `<MakeDir />` task and `tar` will fail the build.

I'm not convinced a comment is necessary but let me know if you're sure we need one.

Reply:
Resolving based on offline conversation&hellip;